packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
+      type: Boolean,
+      default: false,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    withCloseButton: {
+      type: Boolean,
+      default: false,
+    },
+    timeout: {
+      type: Number,
+      default: 8000,
+    },
+    closeCallBack: {
+      type: Function as PropType<Function>,
+      required: true,
+    },
+  },
+  setup(props) {
+    props = reactive(props);
+
+    const handleClose = () => {
+      return props.closeCallBack;
+    };
+
+    return {
+      classes: computed(() => ({
+        'vtmn-toast': true,
+        show: true,
+        [`vtmn-toast--with-icon-info`]: props.withIcon,
+      })),
+      handleClose,
+    };
+  },
+});
+</script>
+
+<template>
+  <div :class="classes" role="status" v-bind="$attrs">
+    <div class="vtmn-toast_content">{{ content }}</div>
```

Comment:
What do you think if we use a slot attribute instead of a props for the content ? It would be easier to control content inside the Toast but maybe this is not a good idea according to the design rules  ?

Reply:
Yes, you're right